[2023-12-20T04:29:39.936+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: service_oriented_class_project.steal_ETL manual__2023-12-20T04:29:38.701815+00:00 [queued]>
[2023-12-20T04:29:39.944+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: service_oriented_class_project.steal_ETL manual__2023-12-20T04:29:38.701815+00:00 [queued]>
[2023-12-20T04:29:39.945+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-20T04:29:39.957+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): steal_ETL> on 2023-12-20 04:29:38.701815+00:00
[2023-12-20T04:29:39.964+0000] {standard_task_runner.py:57} INFO - Started process 18904 to run task
[2023-12-20T04:29:39.967+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'service_oriented_class_project', 'steal_ETL', 'manual__2023-12-20T04:29:38.701815+00:00', '--job-id', '508', '--raw', '--subdir', 'DAGS_FOLDER/service_oriented_class_project.py', '--cfg-path', '/tmp/tmp2ue0a49g']
[2023-12-20T04:29:39.973+0000] {standard_task_runner.py:85} INFO - Job 508: Subtask steal_ETL
[2023-12-20T04:29:40.227+0000] {task_command.py:416} INFO - Running <TaskInstance: service_oriented_class_project.steal_ETL manual__2023-12-20T04:29:38.701815+00:00 [running]> on host f97daee24bd9
[2023-12-20T04:29:40.301+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='christinelee1114@gmail.com' AIRFLOW_CTX_DAG_OWNER='xin' AIRFLOW_CTX_DAG_ID='service_oriented_class_project' AIRFLOW_CTX_TASK_ID='steal_ETL' AIRFLOW_CTX_EXECUTION_DATE='2023-12-20T04:29:38.701815+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-20T04:29:38.701815+00:00'
[2023-12-20T04:29:40.327+0000] {warnings.py:109} WARNING - /opt/***/dags/service_oriented_class_project.py:186: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc.drop('_id', axis=1, inplace=True)

[2023-12-20T04:29:40.328+0000] {warnings.py:109} WARNING - /opt/***/dags/service_oriented_class_project.py:187: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc.rename(columns=newcolumns, inplace=True)

[2023-12-20T04:29:40.332+0000] {warnings.py:109} WARNING - /opt/***/dags/service_oriented_class_project.py:190: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc['日期'] = (df_loc['年'] + 1911).astype(str) + df_loc['月'].astype(str).str.zfill(2) + df_loc['日'].astype(str).str.zfill(2)

[2023-12-20T04:29:40.347+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-20T04:29:40.355+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=service_oriented_class_project, task_id=steal_ETL, execution_date=20231220T042938, start_date=20231220T042939, end_date=20231220T042940
[2023-12-20T04:29:40.381+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-20T04:29:40.544+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
